pathetic-model/src/main/java/org/patheloper/util/ExpiringHashMap.java

```diff
@@ -0,0 +1,57 @@
+package org.patheloper.util;
+
+import java.util.HashMap;
+import lombok.Getter;
+
+public class ExpiringHashMap<K, V> extends HashMap<K, ExpiringHashMap.Entry<V>> {
```

Comment:
Pretty sure this already exists

Reply:
most likely, but its not worth getting an extra dependency for that